DigitalLearningSolutions.Web/Views/TrackingSystem/CentreConfiguration/EditCentreManagerDetails.cshtml

```diff
@@ -12,30 +12,57 @@
 }
 
 @section NavMenuItems {
-  <partial name="~/Views/TrackingSystem/Shared/_NavMenuItems.cshtml"/>
+  <partial name="~/Views/TrackingSystem/Shared/_NavMenuItems.cshtml" />
 }
 
 <div class="nhsuk-grid-row">
   <div class="nhsuk-grid-column-full">
     @if (errorHasOccurred) {
-      <vc:error-summary order-of-property-names="@(new[] { nameof(Model.FirstName), nameof(Model.LastName), nameof(Model.Email), nameof(Model.Telephone) })"/>
+      <vc:error-summary order-of-property-names="@(new[] { nameof(Model.FirstName), nameof(Model.LastName), nameof(Model.Email), nameof(Model.Telephone) })" />
     }
 
-    <h1 id="page-heading" class="nhsuk-heading-xl">Edit centre manager details</h1>

```

Comment:
Is this id for the page heading not used for anything?

Reply:
As far as I can tell, outside of the LearningMenu, no other than messing with the styles by reducing the bottom margin